cli/chain.go

```diff
@@ -1504,3 +1505,38 @@ var ChainPruneCmd = &cli.Command{
 		return api.ChainPrune(ctx, opts)
 	},
 }
+
+var ChainGetTipSetByHeightCmd = &cli.Command{
+	Name:    "gettipset",
+	Aliases: []string{"love"},
+	Usage:   "View a segment of the chain",
+	Flags: []cli.Flag{
+		&cli.Uint64Flag{Name: "height", DefaultText: "current head"},
+	},
+	Action: func(cctx *cli.Context) error {
+		api, closer, err := GetFullNodeAPI(cctx)
+		if err != nil {
+			return err
+		}
+		defer closer()
+		ctx := ReqContext(cctx)
+
+		var head *types.TipSet
+
+		if cctx.IsSet("height") {
+			head, err = api.ChainGetTipSetByHeight(ctx, abi.ChainEpoch(cctx.Uint64("height")), types.EmptyTSK)
+		} else {
+			head, err = api.ChainHead(ctx)
+		}
+		if err != nil {
+			return err
+		}
+
+		head, err = api.ChainGetTipSet(ctx, head.Parents())
```

Comment:
Hmm, I'm not sure I agree with printing info about the parents here. That might match some of our Ethereum-modelling behaviour, but it is just wrong to say the tipset at this height is the parent here.